apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -60,8 +60,8 @@ Page {
                 contentHeight: Theme.itemSizeMedium
```

Comment:
For me it would be clearer if `Default` on line 57 (the text shown in the ComboBox selection when Start View is selected) were changed to `Start view`, so that the text matches the option selected by the user.

There may be some disagreement over this; I tried to figure out the intention from the earlier review discussion but this point remains ambiguous for me.

Reply:
Thanks for making this change, it looks nicer to me.